components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/JsGraphBuilderFactory.java

```diff
@@ -37,5 +37,5 @@
     JsGraphBuilder createBuilder(AuthenticationContext authenticationContext,
                                      Map<Integer, StepConfig> stepConfigMap, AuthGraphNode currentNode);
 
-    ScriptEngine createEngine(AuthenticationContext authenticationContext);
+    Object createEngine(AuthenticationContext authenticationContext);
```

Comment:
Let's not use Object. Let's define a Wrapper class and use it.

Reply:
Resolved with usage of Generic Types